src/content_scripts/applyColorSchemeJs.js

```diff
@@ -6,8 +6,6 @@
 
 let overwroteMatchMedia = false;
 
-const ADDON_FAKED_WARNING = "matchMedia has been faked/overwritten by add-on website-dark-mode-switcher; see https://github.com/rugk/website-dark-mode-switcher/. If it causes any problems, please open an issue.";
```

Comment:
Well, don't just remove it.
IIRC it was there for logging this to the user-visible console, when the API was accessed.

Can't we still log that? It may be useful for debugging when users report issues with the dark theme for web developers, because they e.g. accidentally forget they had this add-on installed. :wink:

Or is this too hard to implement?

Reply:
OK, a message when the hooks are installed.